autostart/lxqt-panel_wayland.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Type=Application
+TryExec=lxqt-panel
+
+# NOTE: KWin wants absolute path here, make sure it's correct
+Exec=/usr/local/bin/lxqt-panel
```

Comment:
This is the reason why only pager worked on arch.
 This file has to be configured by cmake to match the installation path. Changing it to `/usr/bin` kwin_wayland session is fully functional here now :) 
 Could have seen it earlier.